src/main/resources/db/changelog/changes/09-create-cart-items-table.yaml

```diff
@@ -0,0 +1,34 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-cart-items-table
+      author: o.chernyonkov
+      changes:
+        - createTable:
+            tableName: cart_items
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: shopping_cart_id
+                  type: bigint
+                  constraints:
+                    foreignKeyName: fk_shopping_cart_id
+                    references: shopping_carts(id)
+                    nullable: false
+              - column:
+                  name: book_id
+                  type: bigint
+                  constraints:
+                    foreignKeyName: fk_cart_items_book_id
+                    references: books(id)
+                    nullable: false
+              - column:
+                  name: quantity
+                  type: int
+                  constraints:
+                    nullable: false
```

Comment:
Add empty line to each changelogs at the end

Reply:
I added in Order)